lib/db.go

```diff
@@ -0,0 +1,327 @@
+package lib
```

Comment:
Do you think we should create another directory to separate Go and Goby libraries?

Reply:
Yeah that is a good idea. What do you think about having a separate dir for each go package?

Reply:
I think something like
```
lib
  - db
  - other_lib
```
would be great